packages/sitecore-jss/src/graphql-request-client.ts

```diff
@@ -113,7 +113,7 @@ export class DefaultRetryStrategy implements RetryStrategy {
     const rawHeaders = error.response?.headers;
     const delaySeconds = rawHeaders?.get('Retry-After')
       ? Number.parseInt(rawHeaders?.get('Retry-After'), 10)
-      : Math.pow(this.factor, attempt);
+      : Math.pow(this.factor, attempt - 1);
```

Comment:
do we have any tests to ensure delay is correctly calculated? similar to "should use [retry-after] header value when response is 429" case
I think that's what Illia meant by additional tests

Reply:
the last test i added above tests the function getDelay if it return the value correctly.